src/pages/home/top/index.tsx

```diff
@@ -32,10 +32,10 @@ const topData = {
     },
   ],
   versionNote: {
-    text: "Release Note of 2022.0.0.0",
-    link: "https://github.com/alibaba/spring-cloud-alibaba/tree/2022.0.0.0",
+    text: "Release Note of 2023.0.0.0-RC1",
+    link: "https://github.com/alibaba/spring-cloud-alibaba/tree/2023.0.0.0-RC1",
   },
-  releaseDate: "Released on March 22, 2023",
+  releaseDate: "Released on Feb 20, 2024",
```

Comment:
I saw the date is Feb 26, 2024 in [mvn repository](https://mvnrepository.com/artifact/com.alibaba.cloud/spring-cloud-starter-alibaba-nacos-discovery)

Reply:
I just changed it to Feb 26, 2024